test/utils/pickers/describeRangeValidation/testDayViewRangeValidation.tsx

```diff
@@ -67,84 +66,74 @@ export const testDayViewRangeValidation: DescribeRangeValidationTestSuite = (
         testDisabledDate('11', [true, true], !isDesktop || includesTimeView);
       });
 
-      describe('with fake timers', () => {
-        // TODO: temporary for vitest. Can move to `vi.useFakeTimers`
-        let timer: SinonFakeTimers | null = null;
-        beforeEach(() => {
-          timer = useFakeTimers({ now: new Date(2018, 0, 1), toFake: ['Date'] });
```

Comment:
Haven't we already tried removing this time mocking and later encountered edge-cases at certain times? 🤔

Reply:
I can't remember 🤔

I can also keep it in for this describe block if you think it is best

Reply:
I prefer keeping it here. 🤔

Reply:
reverted